code/modules/organs/subtypes/augment.dm

```diff
@@ -348,6 +348,48 @@
 	name = "C'thur language processor"
 	augment_languages = list(LANGUAGE_SKRELLIAN)
 
+/obj/item/organ/internal/augment/gustatoral
+	name = "gustatoral centre"
```

Comment:
I've checked the proper spelling for this and it actually seems to be gustatorial: https://www.yourdictionary.com/gustatorial